frontend_v2/src/app/components/publiclists/teamlist/teamlist.component.ts

```diff
@@ -633,4 +642,32 @@ export class TeamlistComponent implements OnInit, OnDestroy {
     return deleteTeamMember;
   }
 
+  /**
+  * Filter teams by team
+  * @param TeamName Participant team name
+  */
+  filterTeam(teamName) {
+    const SELF = this;
+    SELF.filterTeamsQuery = teamName;
+    let API_PATH = SELF.endpointsService.FilteredParticipantTeamURL(teamName);
```

Comment:
Minor: Can we bring this statement in else case so that it don't have to execute everytime in the case of host team filter.